src/schemas/21-526EZ-allclaims/schema.js

```diff
@@ -0,0 +1,583 @@
+    },
+    otherAtRiskHousing: {
+      type: 'string'
+    },
+    homelessnessContact: {
+      type: 'object',
+      properties: {
+        name: {
+          type: 'string',
+          minLength: 1,
+          maxLength: 100,
+          pattern: "([a-zA-Z0-9-/']+( ?))*$"
+        },
+        phoneNumber: {
+          $ref: '#/definitions/phone'
+        }
+      }
+    },
+    vaTreatmentFacilities: {
+      type: 'array',
+      minItems: 1,
+      maxItems: 100,
+      items: {
+        type: 'object',
+        required: ['treatmentCenterName'],
+        properties: {
+          treatmentCenterName: {
+            type: 'string',
+            maxLength: 100,
+            pattern: "^([a-zA-Z0-9\\-'.#]([a-zA-Z0-9\\-'.# ])?)+$"
+          },
+          treatmentDateRange: {
+            $ref: '#/definitions/dateRangeFromRequired'
+          },
+          treatmentCenterAddress: {
+            $ref: '#/definitions/vaTreatmentCenterAddress'
+          }
+        }
+      }
+    },
+    attachments: {
+      type: 'array',
+      items: {
+        type: 'object',
+        required: ['name', 'attachmentId'],
+        properties: {
+          name: {
+            type: 'string'
+          },
+          confirmationCode: {
+            type: 'string'
+          },
+          attachmentId: {
+            type: 'string',
+            enum: documentTypes526.map(doc => doc.value),
+            enumNames: documentTypes526.map(doc => doc.label)
+          }
+        }
+      }
+    },
+    bankAccountType: {
+      type: 'string',
+      enum: ['Checking', 'Savings']
+    },
+    bankAccountNumber: {
+      type: 'string',
+      minLength: 4,
+      maxLength: 17
+    },
+    bankRoutingNumber: {
+      type: 'string',
+      pattern: '^\\d{9}$'
+    },
+    bankName: {
+      type: 'string',
+      maxLength: 35
+    },
+    isVAEmployee: {
+      type: 'boolean'
+    },
+    standardClaim: {
+      type: 'boolean',
+      default: false
+    },
+    ptsd781Attachment: {
+      $ref: '#/definitions/ptsdAttachment'
+    },
+    ptsd781aAttachment: {
+      $ref: '#/definitions/ptsdAttachment'
+    },
+    form4142: {
+      type: 'object',
+      properties: {
+        limitedConsent: {
+          type: 'string'
+        },
+        providerFacility: {
+          type: 'array',
+          required: [
+            'providerFacilityName',
+            'treatmentDateRange',
+            'providerFacilityAddress'
+          ],
+          items: {
+            type: 'object',
+            properties: {
+              providerFacilityName: {
+                type: 'string'
+              },
+              treatmentDateRange: {
+                $ref: '#/definitions/dateRange'
+              },
+              providerFacilityAddress: {
+                $ref: '#/definitions/address'
+              }
+            }
+          }
+        },
+        privacyAgreementAccepted: {
+          $ref: '#/definitions/privacyAgreementAccepted'
```

Comment:
🤔 that's strange. Looks as though this slipped through on vets-website. I'm going to make it a boolean one-off for now, since the schema's not actually used anywhere.